/monitor_node:
  ros__parameters:
    domain: gazebo
    message_type: Odometry
    min_f: 0.5
    start_type_description_service: true
    topic_name: /odom
    update_f: 1.0
    use_sim_time: false
